client/modules/wj-prism/src/index.ts

```diff
@@ -17,8 +15,18 @@ const encode: (src: string) => string = Prism.util.encode as any
 
 const RAW_LANGS = ["raw", "text", "none", ""]
 
-// add additional langs
-prismFTML(Prism)
+// asynchronously import languages to prevent the large prism JS file
+// from blocking the page from rendering
+// only the base library is synchronously imported and everything else is
+// asynchronously imported
+// prettier-ignore
+async function importLanguages() {
+  ;(await import("../vendor/prism-langs")).prismBase(Prism)
+  ;(await import("../vendor/prism-svelte")).prismSvelte(Prism)
+  ;(await import("./ftml")).prismFTML(Prism)
```

Comment:
why `;(`? why not `;-)`?

Reply:
When you're writing JS without semicolons at the end of the line, it's a convention to stick semicolons at the start of the line. There is probably many ways to argue this, but for me this is helpful because it makes the semicolon really obvious.

Reply:
yeah, I figured, but I wanted to make a dumb joke

Reply:
i'm an idiot lmfao